sources/packages/backend/libs/integrations/src/services/msfaa-number/msfaa-number.service.ts

```diff
@@ -120,6 +153,8 @@ export class MSFAANumberService extends RecordDataModelService<MSFAANumber> {
         serviceProviderReceivedDate: getISODateOnlyString(
           serviceProviderReceivedDate,
         ),
+        cancelledDate: null,
+        newIssuingProvince: null,
```

Comment:
Please include the modifier also in the update.

Reply:
Yes @dheepak-aot , from my understanding the new issuing province should be returned to `null` on msfaa reactivation.

Reply:
`newIssuingProvince` is information that belongs to a canceled MSFAA. As far as I know, there is no reason to keep it for an active MSFAA record. If it can cause confusion we can have it added to the ticket.